src/UNL/UCBCN/Manager/Calendar.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace UNL\UCBCN\Manager;
+
+use UNL\UCBCN\Calendar as CalendarModel;
+use UNL\UCBCN\User;
+
+class Calendar {
+	public $options = array();
+
+	public $calendar;
+
+    public function __construct($options = array()) {
+    	$this->options = $options + $this->options;
+    	$this->calendar = CalendarModel::getByShortName($this->options['calendar_shortname']);
+    }
+
+    public function getCategorizedEvents() {
+    	$events = $this->calendar->getEvents();
```

Comment:
For development, I think this is probably fine, but some sort of limiting is necessary in production.  some calendars have a LOT of events.